docs/content/modules/tensor_mechanics/introduction/Overview.md

```diff
@@ -0,0 +1,124 @@
+
+* An elasticity tensor for crystal plasticity, [ComputeElasticityTensorCP](/Materials/ComputeElasticityTensorCP.md),
+* A Cosserat elasticity tensor [ComputeLayeredCosseratElasticityTensor](/Materials/ComputeLayeredCosseratElasticityTensor.md),
+* An isotropic elasticity tensor [ComputeIsotropicElasticityTensor](/Materials/ComputeIsotropicElasticityTensor.md).  
+
+The input file syntax for the isotropic elasticity tensor is
+
+!text modules/tensor_mechanics/tutorials/basics/part_1.1.i start=elasticity_tensor end=strain overflow-y=scroll max-height=200px
+
+and for an orthotropic material, such as a metal crystal, is
+
+!text modules/tensor_mechanics/tests/finite_strain_elastic/finite_strain_elastic_new_test.i start=elasticity_tensor end=strain overflow-y=scroll max-height=300px
+
+###Stress
+The base class for constitutive equations to compute a stress ($\boldsymbol{\sigma}$) is `ComputeStressBase`. The `ComputeStressBase` class defines the properties `stress` and `elastic_strain`. It is a pure virtual class, requiring all children to override the method `computeQpStress()`.
+
+Two elastic constitutive models have been developed, one that assumes small strains [ComputeLinearElasticStress](/Materials/ComputeLinearElasticStress.md), and a second which assumes finite strains and rotations increments [ComputeFiniteStrainElasticStress](/Materials/ComputeFiniteStrainElasticStress.md) The input file syntax for these materials is
+
+!input modules/tensor_mechanics/tests/finite_strain_elastic/finite_strain_elastic_new_test.i block=stress overflow-y=scroll max-height=500px
+
+There are a number of other constitutive models that have been implemented to calculate more complex elasticity problems, plasticity, and creep.  An overview of these different materials is available on the [Introduction/Stresses](Stresses.md) page.
+
+### Stress-Free Strains (Eigenstrains)
+
+Eigenstrain is the term given to a strain which does not result directly from an applied force. Eigenstrains are also referred to as residual strains, stress-free strains, or intrinsic strains; translated from German, [Eigen](http://dict.tu-chemnitz.de/deutsch-englisch/Eigen....html) means own or intrinsic in English.  The term eigenstrain was introduced by [T. Mura in 1982](http://link.springer.com/chapter/10.1007/978-94-011-9306-1_1):
+
+> Eigenstrain is a generic name given to such nonelastic strains as thermal expansion, phase transformation, initial strains, plastic, misfit strains. Eigenstress is a generic name given to self-equilibrated internal stresses caused by one or several of these eigenstrains in bodies which are free from any other external force and surface constraint.  The eigenstress fields are created by the incompatibility of the eigenstrains.  This new English terminology was adapted from the German "Eigenspannungen and Eigenspannungsquellen," which is the title of H. Reissner's paper (1931) on residual stresses.
```

Comment:
Didn't know we could do this, it looks nice but we may need to reduce the font-size of the quote. It is bigger than the regular text, what do you think? This is something we need to set in the docs/css/moose.css file, not really relevant to this PR.